packages/test-helpers/evmScript.js

```diff
@@ -1,23 +1,31 @@
-const abi = require('ethereumjs-abi')
+const abi = require('web3-eth-abi')
 
-const createExecutorId = id => `0x${String(id).padStart(8, '0')}`
+function createExecutorId(id) {
+  return `0x${String(id).padStart(8, '0')}`
+}
 
-module.exports = {
-  createExecutorId,
+// Encodes an array of actions ({ to: address, calldata: bytes }) into the EVM call script format
+// Sets spec id and concatenates per call
+//   [ 20 bytes (address) ] + [ 4 bytes (uint32: calldata length) ] + [ calldataLength bytes (payload) ]
+// Defaults spec id to 1
+function encodeCallScript (actions, specId = 1) {
```

Comment:
I feel like this logic doesn't belong to a test utility library and we are using it on several places already. Wonder if maybe a few of these across the stack that could belong to a small utility package?

Reply:
Maybe, but it's so small that I don't mind replicating it. At some point it may be worth taking the encode/decode utilities out from aragon.js into a separate package, e.g. `@aragon/contract-toolkit`.